<!--
-->
<launch>
 
  <node name="dll2d_node" type="dll2d_node" pkg="dll2d" output="screen">
    <remap from="/dll2d_node/initial_pose" to="/initialpose"/>
    
    <param name="in_scan" value="/scan" />
    <param name="base_frame_id" value="/base_link" />
    <param name="odom_frame_id" value="/odom" />
    <param name="map_frame_id" value="/map" />
    <param name="update_rate" value="20.0" /> 
    <param name="update_min_d" value="0.2" />
    <param name="update_min_a" value="0.1" />
    <param name="update_min_t" value="1.0" />
    <param name="initial_x"   value="-19.8"/>
    <param name="initial_y"   value="-2.0"/>
    <param name="initial_a"   value="-1.7"/>
    <param name="solver_max_iter"   value="50"/>
    <param name="solver_max_threads"   value="1"/>
    
  </node>

</launch>	
